### DISCUSSION TITLE
Mobile Phone - Impossible to write notes

### DISCUSSION THREAD

**Discussion by Dwindle2234** [TriliumNext/Trilium#3895] [General]
I find creating new notes on my Android phone [Firefox] or Fire Tablet [Silk] impossible.   See attached video where I type "This is a note" twice. I get characters to the right of the cursor.  It seems to be jumping on the predictive suggestions but I can't find a specific pattern - it's random!  It Renders my mobile usage read-only!

Both Mobile and Desktop versions exhibit same issue.  I've never yet had it working.  

Is it me? A setting on my browser?  Or a bug?  Does anyone else have the issue?

https://user-images.githubusercontent.com/120342689/235736842-5f12338f-c163-4986-96da-f855299ee63d.mp4

**Comment by zadam** [comment]
This is an IME issue. Future CKEditor upgrade might help. Alternatively try a different keyboard app.

**Reply by Dwindle2234** [reply]
Thanks. GBoard and SwiftKey no good. Anyone have any suggestions?

**Comment by meichthys** [comment]
@Dwindle2234 I've also experienced this in the past on iOS and Android. 
I recently switched to using the OpenBoard keyboard and haven't noticed the issue (yet)
If it matters, I just tested on vanadium browser with OpenBoard and can't seem to get the issue 🤷‍♂️.

**Reply by Dwindle2234** [reply]
Cheers.  Openboard still has the issue for me unfortunately 😕.

**Reply by meichthys** [reply]
Bummer. Perhaps the browser has something to do with it?
